### PR TITLE
chore(mobile): translation update for russian and ukrainian

### PR DESCRIPTION
**Pull request by OperKH**
- Update `ru` translation
- Update `uk` translation
- Add missed `home_page_no_assets` translation for all translations
- `fr-CA.json` update formatting
- Fix `SharedLinkEditPage` overflow error by replacing `Column` to `ListView`

### CODE REVIEW

File: mobile/assets/i18n/uk-UA.json [hunk 1]
```diff
@@ -121,9 +121,9 @@
...
   "cache_settings_thumbnail_size": "Розмір кешованих мініатюр ({} елементи)",
-  "cache_settings_tile_subtitle": "Control the local storage behaviour",
-  "cache_settings_tile_title": "Local Storage",
-  "cache_settings_title": "Налаштування Кешування",
+  "cache_settings_tile_subtitle": "Керування поведінкою локального сховища",
+  "cache_settings_tile_title": "Локальне сховище",
+  "cache_settings_title": "Налаштування кешування",
   "change_password_form_confirm_password": "Підтвердити пароль",
   "change_password_form_description": "Привіт {name},\n\nВи або або вперше входите у систему, або було зроблено запит на зміну вашого пароля. \nВведіть ваш новий пароль.",
   "change_password_form_new_password": "Новий Пароль",
```
Comment (alex-007) [inline mobile/assets/i18n/uk-UA.json:129]:
```suggestion
  "change_password_form_new_password": "Новий пароль",
```

File: mobile/assets/i18n/uk-UA.json [hunk 2]
```diff
@@ -292,56 +293,56 @@
...
   "profile_drawer_sign_out": "Вийти",
-  "profile_drawer_trash": "Trash",
+  "profile_drawer_trash": "Кошик",
   "recently_added_page_title": "Нещодавні",
-  "scaffold_body_error_occurred": "Error occurred",
+  "scaffold_body_error_occurred": "Виникла помилка",
   "search_bar_hint": "Шукати ваші знімки",
   "search_page_categories": "Категорії",
   "search_page_favorites": "Улюблені",
   "search_page_motion_photos": "Рухомі Знімки",
```
Comment (alex-007) [inline mobile/assets/i18n/uk-UA.json:311]:
```suggestion
  "search_page_motion_photos": "Рухомі знімки",
```

**Review by alex-007 — COMMENTED** [review]
LGTM :+1:

**Comment by alextran1502** [comment]
Hello, we are using Localizely for translation, please send me your email so I can send you the invitation

**Pull request closed without merging**